client/src/store/transaction.js

```diff
@@ -222,6 +241,24 @@ export default {
     async getTransactionInfoByHash({commit}, hash) {
       let transactionInfo = await sdkTransaction.getTransactionInfoByHash(hash)
       commit('setTransactionsInfo', transactionInfo)
+
+      let foramttedTransactionInfo = {
+        transactionId: transactionInfo.transaction.transactionId,
+        timestamp: transactionInfo.timestamp,
+        deadline: transactionInfo.transaction.deadline,
+        status: transactionInfo.status,
+        confirm: transactionInfo.confirm,
+        fee: transactionInfo.transaction.fee,
+        signature: transactionInfo.transaction.signature,
+        signer: transactionInfo.transaction.signer,
+        blockHeight: transactionInfo.transaction.blockHeight,
+        transactionHash: transactionInfo.transaction.transactionHash,
+
+        type: transactionInfo.transaction.transactionBody.type,
+        recipient: transactionInfo.transaction.transactionBody.recipient.address,
+        mosaics: transactionInfo.transaction.transactionBody.mosaics,
```

Comment:
mosaics field is only available with `TransferTransaction`, this is usually grouped under `transaction body`